app/views/content/teaching-internship-providers/_listing.html.erb

```diff
@@ -41,26 +28,33 @@
   <%= render Content::QuoteComponent.new(
     text: "Towards the end of the internship, I was able to teach parts of lessons that I had planned, and I really started to find my feet at the front of the classroom.",
     name: "Ben, former teaching intern",
-    background: "grey",
+    background: "yellow",
     large: false
   ) %>
 </section>
 
-<h2> What you'll cover</h2>
+<h2> What you'll experience</h2>
 
-<p>You'll get a feel for school life, which could include:</p>
+<p>You'll get a feel for what life as a teacher is like. Your internship will give you an opportunity to:</p>
   <ul>
-     <li>shadowing experienced teachers and observing lessons</li>
-     <li>jointly planning and delivering lessons</li>
-     <li>one-to-one guidance from outstanding classroom teachers</li>
-     <li>opportunities to network with qualified subject specialists</li>
+     <li>shadow experienced teachers and observe lessons</li>
+     <li>jointly plan and teach lessons</li>
+     <li>get one-to-one guidance from outstanding classroom teachers and subject specialists</li>
+     <li>see what school life is like for a teacher outside of the classroom</li>
   </ul>
 
 <section class="col col-640">
   <%= render Content::QuoteComponent.new(
     text: "I observed lessons, provided 1:1 support to pupils, and even taught classes! The internship was particularly beneficial in providing a unique insight into teaching training and the application process.",
     name: "Carly, former teaching intern",
-    background: "grey",
+    background: "yellow",
     large: false
   ) %>
 </section>
+
+<h2>Steps to apply</h2>
+<ol>
+  <li>You must apply directly through one of the teaching internship providers below.</li>
```

Comment:
```suggestion
  <li>You must apply directly to one of the teaching internship providers below.</li>
```
This might need confirming with the internships team!